# Число задействованных спутников, [0, inf)
int8 satellites